scripts/genappimage.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+########################################################################
+# Package the baries built on Travis-CI as an AppImage
+# By Simon Peter 2016
+# For more information, see http://appimage.org/
+########################################################################
+
+if [ -z "$ARCH" ]; then
+  export ARCH="$(arch)"
+fi
+
+APP=NVim
+LOWERAPP="${APP,,}"
+ROOT_DIR="$(git rev-parse --show-toplevel)"
+APP_BUILD_DIR="$ROOT_DIR/build"
+APP_DIR="$APP_BUILD_DIR/$APP.AppDir"
+
+# cd neovim
+GIT_REV="$(git rev-parse --short HEAD)"
+
+# Since neovim increments versions slower than vim, using
+# the commit's date makes more sense.
+VIM_VER="$(date -d "@$(git log -1 --format=%ct)" "+%F")"
```

Comment:
Should be NVIM_VER. And why not *first* include a base version (like 0.2.0) and *then* a date and not only a date? Also `git` knows how to format a date: `git show --no-patch --date='format:%F' --format='%cd'`.